dosomething-qa/fastly-frontend/takeover_config.vcl

```diff
@@ -1,8 +0,0 @@
-declare local var.takeover_start STRING;
-declare local var.takeover_end STRING;
-
-# This determines when a timed takeover is applied for this
-# environment (see `shared/takeover_recv.vcl`). Times must
-# be provided in 24-hour EST time.
-set var.takeover_start = "2018-11-07 00:00:00";
-set var.takeover_end = "2018-11-07 14:30:00";
```

Comment:
What's up with these takeover files that were removed, are they just no longer needed?

Reply:
Yep! They were for an old takeover we ran for the 2018 midterm elections.

I'd removed the config that referenced these in [`4fc0a59`](https://github.com/DoSomething/infrastructure/pull/270/files/4fc0a59fec55b0a62df45658bc6b7fd7dd3dda7e#diff-78e34551875d9e35121e22997ec0db55) but forgot to remove the files themselves.